src/pages/guides/index.md

```diff
@@ -243,26 +243,31 @@ To prevent this from happening, Adobe I/O Events has a robust event validation p
 
-**Improved and Resilient Security Verification for Webhook Events**
+### Improved and Resilient Security Verification for Webhook Events
 
 <InlineAlert variant="info" slots="text"/>
 This feature is scheduled to be rolled out on 10th February, 2022
 
 For a more robust and reliable verification, Adobe I/O Events adds below security validations for events delivered to your webhook. 
 
-- sending additional field of recipient_client_id as part of your event payload
-- signing all the event payloads digitally using a fixed public/private key pair generated by Adobe I/O Events and sending those digital signatures as request headers to your webhook
-- sending the relative paths of public keys, which are served from our fixed adobe domain adobeioevents.com so that no one else can use their arbitrary public/private key pairs, as webhook request headers.
+- Adobe I/O Events sends an additional field of `recipient_client_id` as part of your event payload.
+- The event payload is signed digitally using a fixed public/private key pair generated by Adobe I/O Events. The digital signature is sent as a webhook request header.
+- Adobe I/O Events sends the relative path of public key, which is served from our fixed Adobe domain [adobeioevents.com](https://adobeioevents.com), as webhook request headers.
 
-I/O Events generates two public-private key pairs and sign your event payload using digital signatures following the steps below
+**How it strengthens security**
+
+- Adobe I/O Events uses the same public/private key pair generated by itself to sign all event payloads flowing through it's pipeline. The public keys are also hosted at it's own domain i.e. [adobeioevents.com](https://adobeioevents.com). This ensures that any malicious user can't use his own arbitrary public/private key pair to sign an event payload and send forged requests with his random public key url to your webhook endpoint. As, now the webhook consumer is using only the Adobe I/O Events generated public key served via the fixed Adobe domain and provided to him via the request header to verify the signature.
+- Adobe I/O Events also supports custom events. So, consider a scenario that a malicious user `user-A` creates his own custom events, pushes them to Adobe IO Events, receives signed copy of the events and forwards them to `user-B`'s webhook. This can get `User-B` tricked because he can't identify that this event was sent by the malicious user and not Adobe I/O Events. So, he can get such events that he hasn't subscribed for. To avoid this scenario Adobe I/O Events sends the event payload with an additional field of `recipient_client_id` in it. The webhook consumer can match the payload's client id with his own client to verify if he is the actual recipient of the event and that the event is sent by us.
```

Comment:
This is a corner case, and I think we need not explain it in the documentation. We can add a generic line that -
```
Adobe I/O Events sends the event payload with an additional field of `recipient_client_id`. The webhook consumer can match the payload's client id with his own client to verify if he is the actual recipient of the event.
```

Reply:
okay